native-sdk/src/component/package.rs

```diff
@@ -22,7 +22,7 @@ impl BorrowedPackage {
         Y: ClientApi<E>,
     {
         api.call_module_method(
-            RENodeId::GlobalObject(self.0.into()),
+            &RENodeId::GlobalObject(self.0.into()),
```

Comment:
Would rather avoid this reference if `1.68.0` is working.

Reply:
It's not working with `1.68` on `x86_64`.

Why would prefer a copy of RENodeId?

Reply:
CI run: https://github.com/radixdlt/radixdlt-scrypto/actions/runs/4471594998/jobs/7856685400